lms/static/js/learner_dashboard/views/course_card_view.js

```diff
@@ -27,6 +27,7 @@ class CourseCardView extends Backbone.View {
       this.enrollModel.urlRoot = this.urlModel.get('commerce_api_url');
     }
     this.context = options.context || {};
+    this.model.updateCourseRunWithHighestGrade(this.context.courseData.grades);
```

Comment:
I think we should add a check above this, if status is completed otherwise it will set completed course run as active course while rendering remaining and in progress course runs:
```
if (this.context.collectionCourseStatus === 'completed') {
    this.model.updateCourseRunWithHighestGrade(this.context.courseData.grades);
}
```